_includes/workshop_ad.html

```diff
@@ -10,6 +20,11 @@ <h2>{{site.title}}</h1>
       <h2>{{page.venue}}</h2>
       <div class="row">
         <div class="col-md-6">
+          {% if online == "true_public" %}
+          <p><a href="{{site.address}}">Online</a></p>
+          {% elsif online == "true_private" %}
+          <p>Online</p>
+          {% endif %}
```

Comment:
I'm wondering if the advertisement should indicate in the text that an event is public or private so that people don't think that the link is simply broken.

```suggestion
          {% if online == "true_public" %}
          <p><a href="{{site.address}}">Online (click for link to event)</a></p>
          {% elsif online == "true_private" %}
          <p>Online (private)</p>
          {% endif %}
```

Reply:
That's a good point. Maybe we can change the text:
- use "Online" for private events
- use "Online (link)" for public events (where link is clickable).

Below the jumbotron, there is more information that spells out how to access the workshop.

What do you think?

Reply:
I like it!